src/index.d.ts

```diff
@@ -376,6 +385,12 @@ declare namespace RamdaAdjunct {
          */
         ensureArray<T>(value: T | T[]): T[];
 
+        /**
```

Comment:
Don't forget to align this description with the one I proposed in previous comment